package.json

```diff
@@ -2,13 +2,33 @@
   "name": "moduk-frontend",
   "version": "0.1.0",
   "description": "The MOD.UK Frontend contains the code you need to start building a user interface for Ministry of Defence platforms and services.",
-  "main": "dist/index.js",
+  "main": "dist/lib/index.js",
+  "sass": "src/css/index.scss",
+  "style": "dist/css/index.css",
+  "exports": {
+    ".": {
+      "types": "./dist/lib/index.d.ts",
+      "require": "./dist/lib/index.js",
+      "sass": "./src/css/index.scss",
+      "style": "./dist/css/index.css"
+    },
+    "./client": {
+      "import": "./dist/client/MODUK.mjs",
+      "require": "./dist/client/MODUK.umd.js"
+    },
+    "./dist/": "./dist/"
+  },
+  "types": "dist/lib/index.d.ts",
   "scripts": {
-    "build": "npm run build:css",
-    "build:css": "sass --load-path=node_modules --quiet-deps src/css/index.scss dist/index.css",
+    "build": "npm run build:css && npm run build:lib && npm run build:client",
+    "build:client": "vite build && tsc --project src/client",
+    "prebuild:css": "shx rm -rf dist/css",
+    "build:css": "sass --load-path=node_modules --quiet-deps --style compressed src/css/index.scss | postcss -o dist/css/index.css",
+    "prebuild:lib": "shx rm -rf dist/lib",
```

Comment:
When are the prebuild scripts used?  Is it used just to clean dist folder on a dev machine?

Reply:
They run automatically when the non-prefixed script runs (e.g. `prebuild:lib` runs before `build:lib`).

Yes, they're mainly for when working locally as you'd expect to have a clean directory on CI (barring a caching screw-up).